tests/config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = [{
+	deviceType: `raspberrypi3`,
+	suite: `${__dirname}/../suites/os`,
+	config: {
+		networkWired: false,
+		networkWireless: true,
+		downloadType: 'local',
+		interactiveTests: false, // redundant
+		balenaApiKey: process.env.API_KEY,
+		balenaApiUrl: 'balena-cloud.com',
+		organization: process.env.ORG
+	},
+	image: `${__dirname}/balena.img.gz`,
+	workers: {
+		balenaApplication: process.env.BALENA_CLOUD_APP_NAME,
+		apiKey: process.env.BALENA_CLOUD_API_KEY,
+	},
+},
+{
+	deviceType: `raspberrypi3`,
+	suite: `${__dirname}/../suites/hup`,
+	config: {
+		networkWired: false,
+		networkWireless: true,
+		downloadType: 'gunzip',
+		downloadVersion: 'latest',
+		interactiveTests: false, // redundant
+		balenaApiKey: process.env.API_KEY,
```

Comment:
Same here, should be `balenaApiKey: process.env.BALENA_CLOUD_API_KEY`

Reply:
Fixup pushed.